EIPS/eip-155.md

```diff
@@ -65,6 +65,7 @@ This would provide a way to send transactions that work on Ethereum without work
 | 5              | Goerli                                     |
 | 42             | Kovan                                      |
 | 1337           | Geth private chains (default)              |
+| 5777           | Ganache GUI private chains (default)       |
```

Comment:
The rationale for this change is unclear. Why is this necessary? Does Gananche not already implement this?